bigquery/README.md

```diff
@@ -4,3 +4,55 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+## Accessing build insights with SQL queries through the BigQuery CLI:
+
+```bash
+$ bq query '<SQL QUERY>'
+```
+Legacy SQL dialect is set on default for the BigQuery CLI and must be disabled for the example queries to work.
+This can be done by editing ```~/.bigqueryrc``` and adding the following lines:
+
+```
+[query]
+--use_legacy_sql=false
+```
+More information can be found [here](https://cloud.google.com/bigquery/docs/reference/standard-sql/enabling-standard-sql).
```

Comment:
Replace "here" with the "Switching SQL Dialects" (the page title)

Reply:
@Aric1088 please fix.